kodi_addon_checker/__main__.py

```diff
@@ -52,7 +52,7 @@ def check_artifact(artifact_path, args, branch_name, all_repo_addons, pr):
     config = Config(artifact_path, args)
     ConfigManager.process_config(config)
     if os.path.isfile(os.path.join(artifact_path, "addon.xml")):
-        return check_addon.start(artifact_path, branch_name, all_repo_addons, pr, config)
+        return check_addon.start(artifact_path, branch_name, all_repo_addons, pr, not pr, config)
```

Comment:
Not a fan of `sa` as the combination of `pr` and `sa` are not all handled (or available).
Any reason your not using `pr` directly?